### ISSUE TITLE
Simultaneous WebRTC performance degredation in 1.0.1

### ISSUE THREAD

**Issue by trankin** [bluenviron/mediamtx#2281]
## Which version are you using?

v1.0.1

## Which operating system are you using?

- [ ] Linux amd64 standard
- [ ] Linux amd64 Docker
- [ ] Linux arm64 standard
- [ ] Linux arm64 Docker
- [ ] Linux arm7 standard
- [ ] Linux arm7 Docker
- [ ] Linux arm6 standard
- [ ] Linux arm6 Docker
- [ ] Windows amd64 standard
- [ ] Windows amd64 Docker (WSL backend)
- [ ] macOS amd64 standard
- [X] macOS arm64 standard
- [ ] macOS amd64 Docker
- [ ] Other (please describe)

## Describe the issue

There seems to be a bug causing slow / stuttering and often disconnecting streams when loading the same URL in multiple browser windows.. e.g. http://localhost:8889/test1/

Specifically this is for an RTSP to WebRTC stream.  I downgraded back to 1.0.0 and confirmed that it began working as expected again.

## Describe how to replicate the issue
1. start the server
2. publish with an RTSP to WebRTC stream
3. open the same WebRTC Stream in two browser windows.

```
27aec28e-6181-4753-9acd-0456a75f0289:
    runOnDemand: ffmpeg -rtsp_transport tcp -i rtsp://username:password@xxx.xxx.xxx.xxx:554 -acodec pcm_mulaw -vcodec copy -f rtsp rtsp://localhost:$RTSP_PORT/$RTSP_PATH
    runOnDemandRestart: yes
```

[1.0.1 Screen Capture](https://monosnap.com/file/Yb6R6mIfDXSNgmUD3msY666ZUdvYYf)
[1.0.0 Screen Capture](https://monosnap.com/file/fp9hgYcOphJbO4ihLcpVP8fV7UMhbd)

I confirmed the problem also exists in 1.0.2

## Did you attach the server logs?
no

## Did you attach a network dump?
no

**Comment by aler9** [comment]
Thank you very much for reporting this critical issue, a new version is being released immediately (v1.0.3).

**Comment by clee604** [comment]
Not sure if this is a related issue but I'm seeing similar negative behaviour (poor perf, dropouts, etc.) on my rtsp proxy (wyze rtsp -> mediamtx rtsp proxy -> clients even on v.1.0.3.

I rolled back to v.1.0.0 and all seems fine again. I'll need to dig into logs when I have time (it proxies our kid cams) to tinker around with it again.
